extension_methods/example/operator_extensions.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2019, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:extension_methods/some_api.dart';
+
+/// This file shows that classes can be extended even by operators.
```

Comment:
nit: maybe "This file shows that extension methods can be used to overload operators" or "This file shows that extension methods can be used in conjunction with operator overloading".

Reply:
[](url)